runtime/cudaq/platform/default/rest/helpers/equal1/Equal1ServerHelper.cpp

```diff
@@ -82,12 +103,14 @@ void Equal1ServerHelper::initialize(BackendConfig config) {
   if(!equal1ServerURL.ends_with("/"))
     equal1ServerURL += "/";
 
-  machine = getConfig("EQUAL1_TARGET_MACHINE", "machine", "default");
+  machine = getConfig("EQUAL1_TARGET_MACHINE", "machine", "beta2");
```

Comment:
Why is the machine hardcoded here? Isn't there a way to set it like we set the target backend?

Reply:
well, we can set it either from the global config (passed from user) or override it from environment 

That line just sets "beta2" as a default value